.github/workflows/make-live.yml

```diff
@@ -0,0 +1,73 @@
+name: Make Live
+# Make live versions of .Rmd files in training modules
+
+# This workflow only runs when it is manually dispatched
+on:
+  workflow_dispatch:
+    inputs:
+      rendering:
+        description: Should rendering be completed? TRUE or FALSE
+        required: true
+        default: 'FALSE'
+
+jobs:
+  # This workflow contains a single job called "build-all"
+  make-live:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container:
+      image: ccdl/training_dev:latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # get the full repo
+          fetch-depth: 0
+          # use alexslemonade-docs-bot
+          token: ${{ secrets.DOCS_BOT_GITHUB_TOKEN }}
+
+      # Download data
+      #- name: Download data
+      #  run: bash scripts/download-data.sh
+      #- name: Render all pages to html
+      #  run: |
+      #    docker run \
+      #    --mount type=bind,target=/home/rstudio,source=$PWD \
+      #    ccdl/training_dev:latest \
+      #    snakemake --cores 2 --forceall
```

Comment:
I think we should just delete all of this for now, as it is not an accurate representation of what will happen in the actual download step, so having it here as a comment is potentially confusing. 

It we want to have a sketch of this, it might include the `if:` statement to only do the download step if `inputs.rendering` is TRUE.

Reply:
I'll just delete this for a smaller comment since I haven't workshopped this fake code at all yet.